packages/vuetify/src/components/VColorPicker/VColorPicker.tsx

```diff
@@ -117,69 +114,74 @@ export const VColorPicker = defineComponent({
       },
     })
 
-    useRender(() => (
-      <VSheet
-        rounded={ props.rounded }
-        elevation={ props.elevation }
-        theme={ props.theme }
-        class={[
-          'v-color-picker',
-        ]}
-        style={{
-          '--v-color-picker-color-hsv': HSVtoCSS({ ...(currentColor.value ?? nullColor), a: 1 }),
-        }}
-        maxWidth={ props.width }
-      >
-        { !props.hideCanvas && (
-          <VColorPickerCanvas
-            key="canvas"
-            color={ currentColor.value }
-            onUpdate:color={ updateColor }
-            disabled={ props.disabled }
-            dotSize={ props.dotSize }
-            width={ props.width }
-            height={ props.canvasHeight }
-          />
-        )}
-
-        { (!props.hideSliders || !props.hideInputs) && (
-          <div key="controls" class="v-color-picker__controls">
-            { !props.hideSliders && (
-              <VColorPickerPreview
-                key="preview"
-                color={ currentColor.value }
-                onUpdate:color={ updateColor }
-                hideAlpha={ !mode.value.endsWith('a') }
-                disabled={ props.disabled }
-              />
-            )}
-
-            { !props.hideInputs && (
-              <VColorPickerEdit
-                key="edit"
-                modes={ props.modes }
-                mode={ mode.value }
-                onUpdate:mode={ m => mode.value = m }
-                color={ currentColor.value }
-                onUpdate:color={ updateColor }
-                disabled={ props.disabled }
-              />
-            )}
-          </div>
-        )}
-
-        { props.showSwatches && (
-          <VColorPickerSwatches
-            key="swatches"
-            color={ currentColor.value }
-            onUpdate:color={ updateColor }
-            maxHeight={ props.swatchesMaxHeight }
-            swatches={ props.swatches }
-            disabled={ props.disabled }
-          />
-        )}
-      </VSheet>
-    ))
+    useRender(() => {
+      const [sheetProps] = VSheet.filterProps(props)
+
+      return (
+        <VSheet
+          rounded={ props.rounded }
+          elevation={ props.elevation }
+          theme={ props.theme }
+          class={[
+            'v-color-picker',
+          ]}
+          style={{
+            '--v-color-picker-color-hsv': HSVtoCSS({ ...(currentColor.value ?? nullColor), a: 1 }),
+          }}
+          { ...sheetProps }
+          maxWidth={ props.width }
```

Comment:
I think this'll override any maxWidth from props, should that be added to the omit?